docs/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python@sha256:5d769f990397afbb2aca24b0655e404c0f2806d268f454b052e81e39d87abf42
```

Comment:
You need to add the path to this Docker file here https://github.com/makerspace/makeradmin/blob/3b912abfd2cbf2263dd8b4bea6a573ffc9384a68/update-docker-images.sh#L41-L42 so it's also kept up to date by the script

Reply:
Done :)